mappings/net/minecraft/util/SystemDetails.mapping

```diff
@@ -0,0 +1,36 @@
+CLASS net/minecraft/class_6396 net/minecraft/util/SystemDetails
+	FIELD field_33852 MEBIBYTE_DIVIDER J
+	FIELD field_33854 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_33855 OPERATING_SYSTEM Ljava/lang/String;
+	FIELD field_33856 JAVA_VERSION Ljava/lang/String;
+	FIELD field_33857 JAVA_VM_VERSION Ljava/lang/String;
+	FIELD field_33858 sections Ljava/util/Map;
+	METHOD method_37120 toProfilerString ()Ljava/lang/String;
+	METHOD method_37121 tryAddGroup (Ljava/lang/String;Ljava/lang/Runnable;)V
+		ARG 1 name
+		ARG 2 adder
+	METHOD method_37122 addSection (Ljava/lang/String;Ljava/lang/String;)V
+		ARG 1 name
+		ARG 2 value
+	METHOD method_37123 tryAddSection (Ljava/lang/String;Ljava/util/function/Supplier;)V
```

Comment:
Imo `addSection` suffices. There is no clear evidence that this is different from the other `addSection`, as that `String` version cannot throw an exception that needs to be handled.